corehq/apps/domain/models.py

```diff
@@ -146,7 +146,7 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
         choices=['', "plus", "community", "standard", "pro", "advanced", "enterprise"],
         default="community"
     )
-    services = StringProperty(choices=["", "basic", "plus", "full", "custom"], default="")
+    services = StringProperty(choices=["", "basic", "plus", "full", "custom"], default="")  # deprecated
```

Comment:
I was a little scared that these might be used elsewhere. `date` and `services` are pretty hard to search for. I can remove them in a follow up and hope tests pass though